CRM/Contact/BAO/GroupContactCache.php

```diff
@@ -510,7 +510,15 @@ public static function load(&$group, $force = FALSE) {
           );
       }
       $groupID = CRM_Utils_Type::escape($groupID, 'Integer');
-      $sql = $searchSQL . " AND contact_a.id NOT IN (
+
+      if (empty($customClass->where())) {
```

Comment:
TIm and I think you need to use this syntax (your syntax throws this fatal error: ```Can't use method return value in write context in CRM/Contact/BAO/GroupContactCache.php on line 514)```

if (!($customClass->where())) {